stubs/python-decouple/decouple.pyi

```diff
@@ -0,0 +1,168 @@
+from _typeshed import Incomplete
+from collections import OrderedDict
+from collections.abc import Callable, Iterable, Sequence
+from configparser import ConfigParser
+from typing import Any, Generic, Protocol, TextIO, TypeVar, overload
+
+_T = TypeVar("_T")
+_TCsv = TypeVar("_TCsv")
+
+PYVERSION: Incomplete  # undocumented
+DEFAULT_ENCODING: str  # undocumented
+TRUE_VALUES: set[str]  # undocumented
+FALSE_VALUES: set[str]  # undocumented
+
+def read_config(parser: ConfigParser, file: TextIO) -> None: ...  # undocumented
```

Comment:
```suggestion
def read_config(parser: ConfigParser, file: Iterable[str]) -> None: ...  # undocumented
```
Matching https://github.com/python/typeshed/blob/96eaa767ddfb5b206e0a57a5663daf07212b2675/stdlib/configparser.pyi#L150 which this calls. We generally try to avoid the "IO" classes as they are too restrictive.

Reply:
Nice! Updated.